### ISSUE TITLE
[IMPORTANT/BUG] Can't import original modules

### ISSUE THREAD

**Issue by Colk-tech** [Colk-tech/ColkMCServerHelper#14]
The error occurring now is very simple:
```
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from lib.utils import *
ImportError: No module named lib.utils
```
I will fix it imminently.

**Comment by Colk-tech** [comment]
I will fix it in ```hot_fix``` branch.

**Comment by Colk-tech** [comment]
This must be closed according to close of #15 .

**Comment by Colk-tech** [comment]
See here to know how to solve such errors:
https://github.com/Colk-tech/ColkMCServerHelper/pull/15#issue-394388217
